tests/helpers/test_reload.py

```diff
@@ -66,6 +67,11 @@ async def setup_platform(*args):
 
     assert len(setup_called) == 2
 
+    existing_platforms = async_get_platforms(hass, PLATFORM)
+    for existing_platform in existing_platforms:
+        existing_platform.config_entry = "abc"
+    assert not async_get_platform_without_config_entry(hass, PLATFORM, DOMAIN)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)